tests/application/cms/test_views.py

```diff
@@ -727,6 +728,91 @@ def test_dept_user_should_be_able_to_edit_shared_page(test_app_client, logged_in
     assert measure_version.title == "this is the update"
 
 
+@pytest.mark.parametrize("measure_version", ["1.0", "1.1", "2.0"])
+def test_db_version_id_gets_incremented_after_an_update(test_app_client, logged_in_rdu_user, measure_version):
+    measure_version: MeasureVersion = MeasureVersionFactory(
+        status="DRAFT", title="this will be updated", db_version_id=1, version=measure_version
+    )
+
+    response = test_app_client.get(
+        url_for(
+            "cms.edit_measure_version",
+            topic_slug=measure_version.measure.subtopic.topic.slug,
+            subtopic_slug=measure_version.measure.subtopic.slug,
+            measure_slug=measure_version.measure.slug,
+            version=measure_version.version,
+        ),
+        follow_redirects=True,
+    )
+    assert response.status_code == 200
+    page = BeautifulSoup(response.data.decode("utf-8"), "html.parser")
+    assert page.find("input", {"id": "db_version_id"}).attrs["value"] == "1"
+
+    data = multidict_from_measure_version_with_overrides(measure_version, db_version_id="1", title="this is the update")
```

Comment:
Why do we need to override the db_version_id here? Shouldn't it be 1 anyway? Or is that just for clarity in what the test is doing?

Reply:
It's not actually overridden, seeing as the function doesn't add `db_version_id` to the returned dict in the first place. Maybe that's where the `with_overrides` bit is confusing. 🤔

Reply:
You're right, it isn't really useful here. I'll remove it.